dam_autonomy.py

```diff
@@ -0,0 +1,80 @@
+import cv2, time, subprocess
+from pymavlink import mavutil
+from line_follower import LineFollower, Direction
+
+# RC channel IDs (constants)
+RC_CHAN_PITCH = 1
+RC_CHAN_ROLL = 2
+RC_CHAN_THROTTLE = 3
+RC_CHAN_YAW = 4
+RC_CHAN_FORWARD = 5
+RC_CHAN_LATERAL = 6
+
+# mavproxy.py --master=udpin:192.168.2.1:14540 --out=udpout:192.168.2.2:14540
+
+
+def set_rc_channel_pwm(id, pwm=1500):
+    """ Set RC channel pwm value
+
+    Args:
+        id (TYPE): Channel ID
+        pwm (int, optional): Channel pwm value 1100-1900
+    """
+    if id < 1:
+        return
+    if id < 9:
+        rc_channel_values = [65535 for _ in range(8)]
+        rc_channel_values[id-1] = pwm
+        master.mav.rc_channels_override_send(master.target_system, master.target_component, *rc_channel_values)
+
+
+def go_down():
+    set_rc_channel_pwm(RC_CHAN_THROTTLE, 1450)  # half speed, I think
```

Comment:
for this and all other calls to `set_rc_channel_pwm()` you should not have magic numbers.  please take the pwm value and give it a name.  

also you should probably call the method as `set_rc_channel_pwm(RC_CHAN_THROTTLE, pwm=...)` where the ... is the name of the pwm value.  you should do this to the other ones too